images/scripts/rhel.setup

```diff
@@ -296,6 +289,14 @@ EOF
     version=$(cut -d '-' -f 2 <<< "$IMAGE")
     su builder -c "/usr/bin/mock --verbose -i $(/var/lib/testvm/build-deps.sh "$opsys $version")"
     su builder -c "/usr/bin/mock --install --verbose rpmlint"
+
+    # We need to make sure mock has the same selinux-policy as we do
+    if [ "${IMAGE#rhel-9*}" != "$IMAGE" ]; then
+        mkdir /tmp/selinux-policy-rpms
+        dnf download --downloaddir /tmp/selinux-policy-rpms selinux-policy{,-devel,-targeted}
```

Comment:
Will this download the versions that are installed?  Or the ones that would be installed by `dnf update`?  Maybe we need the `dnf udpate selinux-blah` call after all.

Reply:
Arg, I missed the `dnf install` right after this, sorry.